spec/models/cart_spec.rb

```diff
@@ -1,16 +1,37 @@
 require "rails_helper"
 
 RSpec.describe Cart, type: :model do
+  let!(:user) { User.create(email: "example@example.com", password: "password", name: "example") }
 
-  xcontext "with valid attributes" do
-    it "an item is added to the cart" do
-      item = create(:item)
-      cart = Cart.new(nil)
-      cart.add_item(item)
-      expect(cart.cart_items.count).to eq(1)
-      item2 = Item.create
-      cart.add_item(item2)
-      expect(cart.cart_items.count).to eq(2)
-    end
+  let!(:loan_request) { LoanRequest.create(title: "Farm Tools",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.